### PR TITLE
adding a test for filter pipeline description v2

### PR DESCRIPTION
**Pull request by bmaranville**
Here is a test for filter pipeline description version 2, using [a file from the netCDF4 examples](https://github.com/Unidata/netcdf4-python/blob/master/examples/data/prmsl.2011.nc)

### CODE REVIEW

File: tests/test_filter_pipeline_v2.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+""" Unit tests for pyfive's ability to read a file with filter_pipeline v2
+(as is found in some new netCDF4 files) """
+import os
+import warnings
```
Comment (kalvdans) [inline tests/test_filter_pipeline_v2.py]: Unused import, remove

File: tests/test_filter_pipeline_v2.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import numpy as np
+from numpy.testing import assert_array_equal, assert_almost_equal
```
Comment (kalvdans) [inline tests/test_filter_pipeline_v2.py]: Unused imports, remove

Comment (bmaranville) [reply tests/test_filter_pipeline_v2.py]: > 8 MB is a bit on the large side, but better than nothing.

You're right - if I can find a smaller example file that uses the new filter pipeline description I will make a new pull request with that instead.  I tried for a while to create a new file using netCDF4 but was unable to trigger the switch to filter pipeline description v2.

Comment (kmuehlbauer) [reply tests/test_filter_pipeline_v2.py]: @bmaranville 

> I tried for a while to create a new file using netCDF4 but was unable to trigger the switch to filter pipeline description v2.

That would be the way forward, I think.

**Comment by bmaranville** [comment]
I was able to create a small file with filter pipeline description version 2.  I will manually merge this with the feature branch and create a new PR with both the feature and test.

**Comment by kmuehlbauer** [comment]
@bmaranville That sounds great :+1:

**Comment by bmaranville** [comment]
Superseded by #52

**Pull request closed without merging**